src/www/ui/api/documentation/openapi.yaml

```diff
@@ -51,6 +51,7 @@ tags:
 paths:
   /tokens:
     post:
+      deprecated: true
```

Comment:
@krishna9304 I think I missed that new end point requires an existing token to create a new one. If that is the case, this endpoint can not be deprecated then as there will be no way to create token for a user logging in first time then.

Reply:
okay, got it!